sources/Shaarei_Orah/Shaarei_Orah_index_post.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+import sys
+import os
+p = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+sys.path.insert(0, p)
+from sources.local_settings import *
+sys.path.insert(0, SEFARIA_PROJECT_PATH)
+os.environ['DJANGO_SETTINGS_MODULE'] = "local_settings"
+from sources.functions import *
+import re
+import data_utilities
+import codecs
+from sources import functions
+from Shaarei_Orah_parse import *
+
+english_titles= ["Introduction","First Gate, Tenth Sefirah","Second Gate, Ninth Sefirah","Third and Fourth Gates, Seventh and Eight Sefirah","Fifth Gate, Sixth Sefirah","Sixth Gate, Fifth Sefirah","Seventh Gate, Fourth Sefirah","Eight Gate, Third Sefirah","Ninth Gate, Second Sefirah","Tenth Gate, First Sefirah"]
+titles = get_titles()
+text = get_parsed_text()
```

Comment:
These lines are completely unreadable. Since there are multiple `import *` statements, it is impossible to know where the methods come from.